src/tests/JIT/Regression/JitBlue/Runtime_84693/Runtime_84693.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using Xunit;
+
+public class Test
```

Comment:
nit: I typically write a comment about what this test case is trying to verify and what was the underlying problem.